charts/spark-operator-chart/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: spark-operator
 description: A Helm chart for Spark on Kubernetes operator
-version: 1.0.7
-appVersion: v1beta2-1.2.0-3.0.0
+version: 1.0.8
```

```suggestion
version: 1.0.9
```
Since 1.0.8 was released some days ago, I believe this one would be 1.0.9?